internal/files/repository.go

```diff
@@ -24,6 +46,14 @@ func FindRepositoryRootDirectory() (string, error) {
 	for dir != "." {
 		path := filepath.Join(dir, ".git")
 		fileInfo, err := os.Stat(path)
+		if err == nil && !fileInfo.IsDir() {
+			errWorktree := isGitWorktree(path)
+			if errWorktree != nil {
```

Comment:
Nit. This can be `err`.
```suggestion
			err := isGitWorktree(path)
			if err != nil {
```